examples/android/app/BUILD.bazel

```diff
@@ -8,6 +8,9 @@ android_binary(
     incremental_dexing = 0,
     manifest = "src/main/AndroidManifest.xml",
     multidex = "native",
+    manifest_values = {
+        "lib_name": "lib",
```

Comment:
Shouldn't these depend on the binary name rather than a dependency? This feels closer to how it would actually be used - the libraries have their own manifests. Right now you also still have two binaries with the same package name (`app` and `app3`) as they both define this as `lib`.